geometry_msgs/Point position
float64 distance_to_goal
float64 estimated_time_to_goal
bool ready